samza-core/src/main/java/org/apache/samza/execution/JobNodeConfigurationGenerator.java

```diff
@@ -251,7 +251,6 @@ private void configureTables(Map<String, String> generatedConfig, Config origina
             sideInputs.stream()
                 .map(sideInput -> StreamUtil.getSystemStreamFromNameOrId(originalConfig, sideInput))
                 .forEach(systemStream -> {
-                    inputs.add(StreamUtil.getNameFromSystemStream(systemStream));
```

Comment:
minor: can we edit the comment above to just state that marks each side input stream as bootstrap